docs/docs/software-architecture.md

```diff
@@ -59,7 +58,7 @@ So, in the context of our application:
 
 A significant feature of this architecture is the flow of dependencies, which can be seen by the arrows moving in from the blue layer to the yellow layer in figure 1. This signifies that an outer layer can depend on an inner layer, but an inner layer cannot depend on an outer layer. The things that are most likely to change are kept on the outer layers and the things are less likely to change are kept on the inner most layers, helping the application stand the test of time. This makes the inner layers much more stable than the outer layers thus, the tools used to build the application can be modified easily (blue layer) but the core concepts and ideas behind the application are less likely to change (yellow layer). [3]
```

Comment:
helping the app stand the test of time - do you mean to endure the changes that come over time through technology changes etc.?